File: modul-02/PRAK204-2410817310014-INDRA_SURYADILAGA.py
Jari_Jari, Tinggi = map(float,input().split())
Volume = float(3.14 * Jari_Jari * Jari_Jari * Tinggi)
Luas = float(2 * 3.14 * Jari_Jari * (Jari_Jari + Tinggi))
Keliling = float(2 * 3.14 * Jari_Jari)

print(f"Volume   = {Volume:.2f} ")
print(f"Luas     = {Luas:.2f} ")
print(f"Keliling = {Keliling:.2f} ")

File: praktikum-02/PRAK204-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>

int main() {
    float Jari_Jari, Tinggi;
    scanf("%f", &Jari_Jari);
    scanf("%f", &Tinggi);

    float Volume = 3.14 * Jari_Jari * Jari_Jari * Tinggi;
    float Luas = 2 * 3.14 * Jari_Jari * (Jari_Jari + Tinggi);
    float Keliling = 2 * 3.14 * Jari_Jari;
    printf("Volume   = %.2f \n", Volume);
    printf("Luas     = %.2f \n", Luas);
    printf("Keliling = %.2f \n", Keliling );

    return 0;
}